src/Progressstepper/progressstepper.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { PSAProgressstepper } from "./progressstepper";
+
+const meta = {
+  component: PSAProgressstepper,
+  tags: ["autodocs"],
+  parameters: {
+    layout: "centered",
```

Comment:
Can you add `title: "Functional/Stepper"` before the `component:`line, and update layout to `fullscreen`?